apps/src/code-studio/components/AssetUploader.jsx

```diff
@@ -25,7 +27,14 @@ export default class AssetUploader extends React.Component {
    * We've hidden the <input type="file"/> and replaced it with a big button.
    * Forward clicks on the button to the hidden file input.
    */
-  fileUploadClicked = () => this.refs.uploader.openFileChooser();
+  fileUploadClicked = () => {
+    this.refs.uploader.openFileChooser();
+    analyticsReporter.sendEvent(
+      EVENTS.UPLOAD_CUSTOM_IMAGE,
+      {LabType: 'applab'},
```

Comment:
Noting that AssetUploaded is also used by Web Lab and Java Lab so 'applab' LabType is not accurate here. Informed product already.